shared/Button.tsx

```diff
@@ -19,15 +19,15 @@ interface ToggleButtonProps {
   value?: boolean
 }
 
-export const Button: FC<ButtonProps> = ({ children, className, onClick, ...rest }) => {
+export const Button = ({ children, className, onClick, ...rest }: ButtonProps) => {
   const click = () => {
```

Comment:
Why did we remove `FC<ButtonProps>`?

Reply:
While fixing issues style flipping, it will be removed.